packages/e2e-tests/test/e2e-current-op.spec.ts

```diff
@@ -0,0 +1,82 @@
+import { expect } from 'chai';
+import {
+  skipIfApiStrict,
+  startSharedTestServer,
+} from '../../../testing/integration-testing-hooks';
+import type { TestShell } from './test-shell';
+
+describe('e2e currentOp', function () {
+  skipIfApiStrict();
+
+  const testServer = startSharedTestServer();
+
+  context('with 2 shells', function () {
+    let helperShell: TestShell;
+    let currentOpShell: TestShell;
+
+    const OPERATION_TIME = 1_000;
```

Comment:
Could this be smaller without causing issues? 
```suggestion
    const OPERATION_TIME = CURRENT_OP_WAIT_TIME * 2;
```